src/cmd/logic-function.js

```diff
@@ -20,25 +20,34 @@ module.exports = class LogicFunctionsCommand extends CLICommandBase {
 		super(...args);
 	}
 
-	async list({ org }) {
+	async list({ org, display = true }) {
```

Comment:
This sounds like we want to have a `display` argument in the command.
I think this is not a good approach for that (we're mixing responsibilities over here)
Put a TODO to fix it by creating a list function that returns the thing we want to return in this case I think the goal is return a `choice`  object.